python/grass/temporal/abstract_space_time_dataset.py

```diff
@@ -2579,6 +2579,14 @@ def update_from_registered_maps(self, dbif=None):
         sql = sql.replace("SPACETIME_REGISTER_TABLE", stds_register_table)
         sql = sql.replace("SPACETIME_ID", self.base.get_id())
 
+        # check version
+        idx = sql.find("-- VERSION")
+        if idx > -1:
+            from .core import get_tgis_db_version_from_metadata
+            version = int(sql[idx+len("-- VERSION ")])
+            if get_tgis_db_version_from_metadata() < version:
+                sql = sql[:idx] # remove version-relevant code
```

Comment:
This looks good. But can you please document somewhere (here or in the SQL file) what is the intended behavior and/or how to use it? ...e.g., in case we need to add one more thing like this.

Reply:
Please consider these changes as a draft. We have to decide whether:

1) keep all SQL statements in one file and introduce VERSION specific code by comments (as used in the draft - but by robust and well-documented code) or
2) we split version specific SQL statements into separated files, eg.

* `update_strds_metadata_template.sql` (all versions)
* `update_strds_metadata_template_v3.sql` (version 3+)

The second options is probably more clear and robust. @wenzeslaus What do you think?

Reply:
+1 for the filenames. Having a convention for filenames is also more expected behavior/code than special comments in this case.

Reply:
Agree, I will update this PR accordigly.

Reply:
Done 4a2939757.